lib/loggly/client.js

```diff
@@ -23,6 +23,17 @@ function stringify(msg) {
   return payload;
 }
 //
+// function to truncate message over 1 MB
+// 
+function truncateLargeMessage(message) {  
+  var MaximumBytesAllowedToLoggly = 1000 * 1000;
```

Comment:
@mchaudhary, Actually I hard coded value of  `MaximumBytesAllowedToLoggly` because we can send 1 MB per event i.e. maximum `1048576 bytes` and in different encoding  techniques like `utf-8, utf-16` etc, individual character can take `1 to 6 bytes` of memory space that increases the log message size each time. Due to this reason I took a safe side value less than actually accepted value so that if log message size increases due to different encoding techniques our final log message size would be less than 1 MB after truncation and will also reach to loggly successfully. I have also tested with different payload size and this logic was successfully truncating log message greater than 1 MB and logs were also reaching to loggly of less than 1 MB event size. 
In that case we can not make `MaximumBytesAllowedToLoggly` configurable/dynamic. Anyways this logic is only applicable for truncating log message greater than 1 MB and not affecting any other code in library. 
I have updated code to set value of `MaximumBytesAllowedToLoggly` using a constant variable.